src/cmps/common/ConfettiAnimation.vue

```diff
@@ -17,14 +17,30 @@
 </template>
 
 <script>
-export default {}
+export default {
+  data() {
+    return {
+      isAnimationstarted: false,
```

Comment:
isAnimationstarted should be isAnimationStarted

Reply:
fixed